project_sequence/models/project_project.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2023 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+
+from odoo import api, fields, models
+
+
+class ProjectProject(models.Model):
+    _inherit = "project.project"
+
+    codesequence = fields.Char(
```

Comment:
Let's give this field a more natural name.

```suggestion
    sequence_code = fields.Char(
```